wait_for.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
@parth-gr can we move this `tests/github_action_helper` file as a function?

Reply:
DOne